settings.py

```diff
@@ -1186,18 +1192,16 @@ def JINJA_CONFIG():
         'kuma': {
             'handlers': ['console'],
             'propagate': True,
-            # Use the most permissive setting. It is filtered in the handlers.
-            'level': logging.DEBUG,
+            'level': logging.ERROR,
         },
         'django.request': {
             'handlers': ['console'],
             'propagate': True,
-            # Use the most permissive setting. It is filtered in the handlers.
-            'level': logging.DEBUG,
+            'level': logging.ERROR,
         },
         'elasticsearch': {
-            'level': logging.ERROR,
             'handlers': ['console'],
+            'level': logging.ERROR,
```

Comment:
We've talked to webops but it always didn't work out. With cloudops that was promised to be different.